$OpenBSD: patch-src_Darcs_Util_Encoding_IConv_hsc,v 1.2 2015/12/30 22:04:11 kili Exp $

Don't assume nl_items ((from nl_types.h) is 32 bit everywhere.

http://bugs.darcs.net/patch1416

--- src/Darcs/Util/Encoding/IConv.hsc.orig	Mon Nov  9 14:53:00 2015
+++ src/Darcs/Util/Encoding/IConv.hsc	Wed Dec 30 22:45:31 2015
@@ -37,7 +37,8 @@ import Foreign
     , with, maybeWith
     , ForeignPtr, withForeignPtr, newForeignPtr
     , FunPtr
-    , Int32, Word8
+    , #type nl_item
+    , Word8
     )
 import Control.Exception ( bracket )
 import Data.ByteString ( ByteString, useAsCStringLen, append )
